apps/web/features/feedback-panel/feedback-panel-details.tsx

```diff
@@ -0,0 +1,58 @@
+import { SentimentAnalysisResult } from "@/types/sentiment-analysis-result";
+import { formatCreatedAtDate } from "@/utils/date-utils";
+import FeedbackBadge from "@/components/user-feedback/feedback-badge";
+import ErrorUnexpected from "@/features/error-messages/error-unexpected";
+import FeedbackPanelSkeleton from "@/features/feedback-panel/feedback-panel-skeleton";
+
+type Props = {
+  feedback?: SentimentAnalysisResult;
```

Comment:
It isn't very clear from a signature perspective. You mark this prop as optional, but this prop is required for the component to operate.

It's better to list possible types in this case e.g., `feedback: SentimentAnalysisResult | null | undefined`.

Reply:
Oh, it makes sense. Other devs might ignore it while using with optional param.